defaults/main.yml

```diff
@@ -4,6 +4,8 @@ redis_version: 2.8.24
 redis_install_dir: /opt/redis
 redis_dir: /var/lib/redis/{{ redis_port }}
 redis_download_url: "http://download.redis.io/releases/redis-{{ redis_version }}.tar.gz"
+
+redis_protected_mode: yes
```

Comment:
this should be "yes" not yes, otherwise the .conf.j2 will write True, which will fail to start